pkg/backend/telemetryapi/request.go

```diff
@@ -33,17 +33,36 @@ type requestsBuilder interface {
 }
 
 var (
-	errUnableToSplit = fmt.Errorf("unable to split large payload further")
+	errUnableToSplit     = fmt.Errorf("unable to split large payload further")
+	maxEntitiesByRequest = 100
```

Comment:
Could we move `maxEntitiesByRequest` to the [telemetryapi config.go](https://github.com/newrelic/infrastructure-agent/blob/master/pkg/backend/telemetryapi/config.go) and set it up so when we build the client [here](https://github.com/newrelic/infrastructure-agent/blob/76f487e10e33df62c4137e24762585c677cb3c14/pkg/integrations/v4/dm/telemetry.go#L48) we use the value from [agent config.go](https://github.com/newrelic/infrastructure-agent/blob/master/pkg/config/config.go).

Reply:
What would be the reason on having `maxEntitiesByRequest` as a configurable setting?

Reply:
Done!
(This change will give us flexibility in case `maxEntitiesByRequest` has to change. We can request the customer to adjust that number by configuration in the meantime)

Reply:
Let me know what do you think about naming... I have my doubts. TY ~~